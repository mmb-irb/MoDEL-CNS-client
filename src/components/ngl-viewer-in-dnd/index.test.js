import React from 'react';
import { shallow } from 'enzyme';

import NGLViewerSpawner from '.';
// Esto ha cambiado mucho. hay que probarlo
describe('<NGLViewerSpawner />', () => {
  it('should render', () => {
    expect(
      shallow(
        <NGLViewerSpawner
          accession="MCNS00001"
          analysis="fluctuation"
          requestedFrame="1"
        />,
      ),
    ).toMatchSnapshot();
  });
});
