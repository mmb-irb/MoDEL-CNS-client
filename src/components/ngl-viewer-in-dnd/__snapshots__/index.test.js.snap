// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`<NGLViewerSpawner /> should render 1`] = `
<NGLViewerInDND
  accession="MCNS00001"
  analysis="fluctuation"
  close={[Function]}
  id={0}
  key="0"
  nail={[Function]}
  requestedFrame="1"
/>
`;
